app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Gathering school direct contact details from higher education institutions
+description: Using a spreadsheet to help higher education institutions invite their school direct partners to use  Manage
+date: 2021-04-08
+---
+
+We’ve been looking at ways to help higher education institutions (HEIs) to:
+
+- set themselves up to use Manage
+- invite their school direct partners (SDs) to set themselves up
+
+We started by designing a [small online service that allowed HEIs to set themselves up and invite SDs](https://bat-design-history.netlify.app/manage-teacher-training-applications/self-service-registration/). 
+
+After testing that design we decided to explore a simpler approach, focusing on gathering contact details for SDs.
+
+## What we did and why
+
+We created a spreadsheet template that we could send to HEIs as a CSV file. The columns in the template were:
+
+- partner (pre-filled with the name of an SD) 
+- code (pre-filled with the SD’s provider code)
+- contact name (to be completed by the HEI user)
+- contact email (to be completed by the HEI user)
+
+We decided to keep the headings short as we have no control over column width in a CSV file. When we tested it using Microsoft Excel, the default column width was very narrow.
+
+If an SD was already using Manage, the contact name and email cells would be pre-filled with ‘Not needed - already has account’. 
+
+We also created an email explaining what we wanted users to do. It asked them to either: 
+
+- download the spreadsheet, complete it and email it back
+- fill in a Google Sheets version of the spreadsheet online
+
+The email said that courses would become available as soon as the contact at the SD set up their account.
```

Comment:
Courses become available in what sense? Do providers/SDs still need to do something to start receiving applications? I think they need to update the Published courses. Or maybe the Transition team need to press a button...

Reply:
In this part of the design history entry I'm just reporting what we said in the email, which was vague about what would happen. This was largely because we wanted to prompt a conversation about it with users.

As I understand it, the courses will instantly become available on Apply if they're already available through UCAS. I think users would only need to do something in Publish if the courses were not already available in UCAS. I'll check though, and clarify later in the entry.